lxd/identities.go

```diff
@@ -516,6 +517,18 @@ func tlsIdentityTokenValidate(ctx context.Context, s *state.State, token api.Cer
 		return uuid.UUID{}, fmt.Errorf("Failed to find a matching pending identity: %w", err)
 	}
 
+	reverter := revert.New()
+	defer reverter.Fail()
+
+	reverter.Add(func() {
+		err := s.DB.Cluster.Transaction(s.ShutdownCtx, func(ctx context.Context, tx *db.ClusterTx) error {
+			return dbCluster.DeleteIdentity(ctx, tx.Tx(), id.AuthMethod, id.Identifier)
+		})
+		if err != nil {
+			logger.Warn("Failed to delete invalid or expired pending TLS identity", logger.Ctx{"err": err, "identity_id": id.Identifier})
```

Comment:
can we just use `"identity":` in the log context?

Have you logged an identifier ID previously, what context label did you use?

Reply:
I did not find any other occurences of logging an identifier ID, so just using `identity` should be fine.